changelog

```diff
@@ -2,6 +2,7 @@ MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-andr
 
 vNext
 ----------
+- [MINOR] Update minSDK to 21
```

Comment:
MAJOR?

Reply:
not sure either, considering this will leave some android versions out I will put it as major